### PR TITLE
Manifest

### PR DESCRIPTION
**Pull request by wandyezj**

### CODE REVIEW

File: src/blocks.ts [hunk 1]
```diff
@@ -2,15 +2,15 @@ import { getDataForTable } from "./xl/data";
...
+// export type DataTable = any;
 
-export let currentWorkspace;
+export let currentWorkspace: unknown;
 
 export const blocks = [
     {
         kind: "block",
-        type: "excel_import_table",
+        type: "excelImportTable",
```
Comment (pelikhan) [inline src/blocks.ts]: don't rename these identifiers, they are used in the serialized blocky and will break existing files

**Review by pelikhan — CHANGES_REQUESTED** [review]

File: src/xl/tables.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-export const DATA_SCIENCE_EXPORT_SHEET = "Data Science Editor Export";
-export const DATA_SCIENCE_EXPORT_TABLE = "DataScienceEditorTable";
+export const dataScienceExportSheet = "Data Science Editor Export";
```
Comment (pelikhan) [inline src/xl/tables.ts]: I like to capitalize constants, call me old school. Disable the rule.

**Review by pelikhan — APPROVED** [review]
LGTM aside from the block id changes. Block ids are dash in DS anyway so let's keep it that way.

File: src/blocks.ts (hunk 1 above)
Comment (wandyezj) [reply src/blocks.ts]: Added a note.

File: src/xl/tables.ts (hunk 2 above)
Comment (wandyezj) [reply src/xl/tables.ts]: fixed.

**Pull request merged**